core/src/main/java/com/backblaze/b2/client/credentialsSources/B2CredentialsFromEnvironmentSource.java

```diff
@@ -76,7 +76,7 @@ private String getApplicationKeyIdOrThrow() throws IllegalArgumentException{
             // maybe fallback to using the old variable
             final String accountId = System.getenv("B2_ACCOUNT_ID");
             if (accountId != null) {
-                B2Preconditions.checkState( accountId.isEmpty(),
+                B2Preconditions.checkState( !accountId.isEmpty(),
```

Comment:
good catch.

Reply:
Can you pull this out of this PR and make a new one with just this change?

Reply:
nvm ill do it

Reply:
@spartan218 just merged this fix into master because it's important.  :)